### ISSUE TITLE
Where Can One Explore / Play with a FedWiki (Sandbox Disappeared)

### ISSUE THREAD

**Issue by finneycanhelp** [WardCunningham/Smallest-Federated-Wiki#192]
I used to be able to go to an FAQ page which would lead me to a sandbox to enjoy and spark the imagination about what it would be like to use a SFW. 

It seems that in order to just try out wiki post submissions, I have to "Install and Launch" as described at https://github.com/WardCunningham/Smallest-Federated-Wiki  or do as "Hosting and Installation Guide" states at https://github.com/WardCunningham/Smallest-Federated-Wiki/wiki/Hosting-and-Installation-Guide

In short, where can one just "play" for 6 minutes?  Thanks!
